package.json

```diff
@@ -30,7 +30,7 @@
     "tslint": "tslint",
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
     "sync-externals": "npm run-script sync-schema && npm run-script sync-colors",
-    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data && rm -rf node_modules",
+    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/desired_unread_api && yarn install && npm run dump-schema -- ../../data && rm -rf node_modules",
```

Comment:
FYI The dump-schema script lives in the MP repo, so you can always just run it from your local dev checkout and specify the path to your local Emission checkout.